packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -165,7 +165,18 @@ export function getAuthHeaders(request) {
 			authorization: `Bearer ${oauth_token}`,
 		};
 	}
+
+	// Cookie + CSRF auth: need have matching UUID in MEETUP_CSRF cookie and 'csrf-token' header
 	const cookies = { ...request.state };
+	const csrfCookie =
+		process.env.NODE_ENV === 'production'
+			? cookies.MEETUP_CSRF
+			: cookies.MEETUP_CSRF_DEV;
+	if (csrfCookie) {
```

Comment:
I didn't look too closely at this... but is it possible that the cookie can expire before this happens?

Reply:
in this case, if the cookie expires, it won't be sent in the `request` - this is just a relay of what the browser sends, so we shouldn't have to check any other metadata